packages/integrations/prefetch/README.md

```diff
@@ -78,6 +80,30 @@ export default defineConfig({
   })],
 });
 ```
+
+### config.intentSelector
+By default, the prefetch script also searches the page for any links that include a `rel="prefetch-intent"` attribute, ex: `<a rel="prefetch-intent" />`. This behavior can be changed in your astro.config.* file to use a custom query selector when finding prefetch-intent links.
+
+Importantly, the intentSelector accepts both strings and arrays, giving you the option to prefetch multiple selectors on hover if you want.
+
+__`astro.config.mjs`__
+
+```js
+import { defineConfig } from 'astro/config';
+import prefetch from '@astrojs/prefetch';
+
+export default defineConfig({
+  // ...
+  integrations: [prefetch({
+    // Only prefetch links with an href that begins with `/products` or `/coupons`
+    intentSelector: ["a[href^='/products']", "a[href^='/coupons']"]
+
+    // Below is also valid syntax
+    // intentSelector: "a[href^='/products']"
```

Comment:
```suggestion
     // Only prefetch links with an href that begins with `/products` or `/coupons`
    intentSelector: ["a[href^='/products']", "a[href^='/coupons']"]
    
    // Use a string to prefetch a single selector
    // intentSelector: "a[href^='/products']"
```
I think this is a bit more illustrative than the previous, and means we can avoid a sentence!